rules/windows/persistence_startup_folder_scripts.toml

```
@@ -1,16 +1,61 @@
 [metadata]
 creation_date = "2020/11/18"
 maturity = "production"
-updated_date = "2021/03/03"
+updated_date = "2022/03/21"
 
 [rule]
 author = ["Elastic"]
-description = "Identifies script engines creating files in the startup folder, or the creation of script files in the startup folder."
+description = """
+Identifies script engines creating files in the startup folder, or the creation of script files in the startup folder.
+Adversaries may abuse this technique to maintain persistence in an environment.
+"""
 from = "now-9m"
 index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Persistent Scripts in the Startup Directory"
+note = """## Triage and analysis
+
+### Investigating Persistent Scripts in the Startup Directory
+
+Windows Startup folder is a special folder in Windows. Programs added to this folder are executed during the logon of an
+account, without user interaction, making it an excellent opportunity for attackers to maintain persistence.
+
+This rule will monitor for shortcuts created by wscript.exe or cscript.exe or js/vbs scripts created by any process.
+
+#### Possible investigation steps:
+
+- Investigate process execution chain (parent process tree).
+- Investigate other alerts related to the user/host in the last 48 hours.
+- Validate the activity is not related to planned patches, updates, network administrator activity or legitimate software
+installations.
+- Determine if activity is unique by validating if other machines in same organization have similar entry.
+- Retrieve the script file:
+  - Use a sandboxed malware analysis system to perform analysis.
+    - Observe attempts of contacting external domains and addresses.
+  - Use the PowerShell Get-FileHash cmdlet to get the SHA-256 hash value of the file.
+    - Search for the existence of this file in resources like VirusTotal, Hybrid-Analysis, CISCO Talos, Any.run, etc.
+
+### False Positive Analysis
+
+- There is a low possibility of benign legitimate scripts being added to Startup folders. Validate whether this activity
+is benign.
+
+
+### Related Rules
+
+- Suspicious Startup Shell Folder Modification - c8b150f0-0164-475b-a75e-74b47800a9ff
+- Startup Folder Persistence via Unsigned Process - 2fba96c0-ade5-4bce-b92f-a5df2509da3f
+
+### Response and Remediation
+
+- Initiate the incident response process based on the outcome of the triage.
+- Isolate the involved host to prevent further post-compromise behavior.
+- If the triage identified malware, search the environment for additional compromised hosts.
+  - Implement any temporary network rules, procedures, and segmentation required to contain the malware.
+  - Immediately block the IoCs identified.
```

```suggestion
  - Immediately block external network domains and addresses communicating with malware if optional.
```

---

Depending on [the discussion above](https://github.com/elastic/detection-rules/pull/1871/files#r833637623), if we're staying with "IoCs" I suggest spelling out the term first:
```suggestion
  - Immediately block the identified indicators of compromise (IoCs).
```